app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :update]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create(game_params.merge(white_player_id: current_user.id))
+    if @game.persisted?
+      redirect_to game_path(@game)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    begin 
```

Comment:
Informational: Ruby methods implicitly act like a `begin` block, so you don't need to use `begin` to rescue from errors thrown anywhere in the method: http://seejohncode.com/2012/04/17/short-tip-rescuing-a-method-in-ruby/